pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -274,10 +262,22 @@ RUN rm -rf /root/.ssh/ && \
 
 RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
 
-COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
-RUN chmod +x /usr/local/bin/deep_learning_container.py
+# Install Nvidia Apex (needs pytorch)
+## Pin apex commit requested by sm-model-parallel team
+RUN git clone https://github.com/NVIDIA/apex \
+  && cd apex \
+  && git checkout aa756ce \
```

Comment:
I assume we ping the Apex version because there is an issue for Rubik? @sallyseok could you provide inputs here? I remember we need a fix for it to work with torch 2.0.

Make sure this commit includes the fix for 2.0
https://github.com/NVIDIA/apex/issues/1046

Reply:
Yes, yongyang request to revert to this commit.